tests/components/eheimdigital/__init__.py

```diff
@@ -0,0 +1 @@
+"""Tests for the EHEIM Digital integration."""
```

Comment:
The config flow must have complete code coverage.

See example:
https://github.com/home-assistant/core/blob/dev/tests/components/cambridge_audio/test_config_flow.py